cassovary-core/src/main/scala/com/twitter/cassovary/graph/Traverser.scala

```diff
@@ -335,7 +335,7 @@ trait QueueBasedTraverser[+V <: Node] extends Traverser[V] {
       degreeLimit.isLimitReached(node.neighborCount(dir))) {
       Seq()
     } else {
-      limitAddedToQueue(node.neighborIds(dir).filter(n => shouldBeEnqueued(coloring.get(n))))
+      limitAddedToQueue(node.neighborIds(dir).toSeq.filter(n => shouldBeEnqueued(coloring.get(n))))
```

Comment:
Not really
1) filter
As we don't have ArrayBuffer, we would have to copy filtered elements at least twice.
2) map
if we wanted CSeq[Int].map[T](f: Int => T): CSeq[T]
  If we don't have CSeq[T] that is efficiently implemented we loose the efficiency.
if we wanted CSeq[Int].map(f: Int => Int): CSeq[Int] we are not consistent with scala's map API and this version doesn't give as much.